catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -183,12 +163,40 @@ public Metacard transform(InputStream input, String id)
     }
 
     if (id != null) {
-      metacard.setId(id);
+      metacard.setAttribute(new AttributeImpl(Metacard.ID, id));
     }
 
     return metacard;
   }
 
+  protected Metacard getBaseMetacard(
+      Map<String, Object> properties, List<MetacardType> metacardTypes)
+      throws CatalogTransformerException {
+    final String propertyTypeName = (String) properties.get(METACARD_TYPE_PROPERTY_KEY);
+    MetacardImpl metacard;
+
+    if (isEmpty(propertyTypeName) || metacardTypes == null) {
+      LOGGER.debug(
+          "MetacardType specified in input is null or empty.  Assuming default MetacardType");
+      metacard = new MetacardImpl();
```

Comment:
:pencil2: You could just `return new MetacardImpl();`, and similar approach can be used on line 197.